src/test/java/org/apache/maven/plugins/resources/stub/MavenProjectBuildStub.java

```diff
@@ -300,8 +293,6 @@ private void populateFile( File file ) throws IOException
             try ( FileOutputStream outputStream = new FileOutputStream( file ) )
             {
                 outputStream.write( data.getBytes() );
-                outputStream.flush();
```

Comment:
Don't think the `.flush()` is guaranteed by try-with-resources... but perhaps by not wrapping FOS in a BufferedOutputStream then that's not a problem?

Reply:
close also flushes

Reply:
Calling flush just before close is generally useless cause close ensures all data have been written.
That said we are speaking of a FileOutputStream where flush() is a noop so it is even safer to drop.